MediaBrowser.Controller/Lyrics/LyricMetadata.cs

```diff
@@ -0,0 +1,54 @@
+using System;
+
+namespace MediaBrowser.Controller.Lyrics;
+
+/// <summary>
+/// LyricMetadata model.
+/// </summary>
+public class LyricMetadata
+{
+    /// <summary>
+    /// Gets or sets Artist - The song artist.
+    /// </summary>
+    public string? Artist { get; set; }
+
+    /// <summary>
+    /// Gets or sets Album - The album this song is on.
+    /// </summary>
+    public string? Album { get; set; }
+
+    /// <summary>
+    /// Gets or sets Title - The title of the song.
+    /// </summary>
+    public string? Title { get; set; }
+
+    /// <summary>
+    /// Gets or sets Author - Creator of the lyric data.
+    /// </summary>
+    public string? Author { get; set; }
+
+    /// <summary>
+    /// Gets or sets Length - How long the song is.
```

Comment:
In what unit? Ticks? Milliseconds? (I suspect milliseconds looking at the offset property)

Reply:
Updated the summary.